File: java-apps/kq-common/src/main/java/com/kafkaquest/kq/common/util/models/events/other/TransportScoreCard.java
package com.kafkaquest.kq.common.util.models.events.other;

import com.kafkaquest.kq.common.util.models.events.enums.Grade;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class TransportScoreCard {
    private Long timeSpent;
    private Grade grade;
}

File: kq-projection-dotnet/kq-projection/Common/Models/Jigsaw/TransportScoreCard.cs
namespace KQ_Projection.Common.Models.Jigsaw;
using KQ_Projection.Common.Enums;
using System.Text.Json.Serialization;

public class TransportScoreCard
{
    [JsonPropertyName("timeSpent")]
    public long timeSpent { get; set; }

    [JsonPropertyName("grade")]
    [JsonConverter(typeof(JsonStringEnumConverter))]
    public Grade grade {get; set;}
}
